ui/perfherder/graphs/GraphsContainer.jsx

```diff
@@ -283,7 +283,7 @@ class GraphsContainer extends React.Component {
           }`}
           ref={this.tooltip}
         >
-          <Button className="close mr-3 my-2 ml-2" onClick={this.closeTooltip}>
+          <Button className="mr-3 my-2 ml-2" close onClick={this.closeTooltip}>
```

Comment:
Nice - I didn't realize there was a prop for that. I'd just be sure it looks the same - whatever we use we want it to be styled consistently.

Reply:
Yes, I didn't know it neither. Calling "close" inside the class was actually causing the style diversion. So I've tried this way and it kept the previous style.